lib/LoginScreen.dart

```diff
@@ -20,28 +20,25 @@ class _LoginScreenState extends State<LoginScreen> {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      debugShowCheckedModeBanner: false,
-      home: Scaffold(
-        backgroundColor: Color(0xfff8f9fa), // just add oxff before hexa code
-        body: SingleChildScrollView(
-          child: SafeArea(
-            child: Column(
-              //      mainAxisAlignment: MainAxisAlignment.start,
-              // crossAxisAlignment: CrossAxisAlignment.center,
-              children: <Widget>[
-                Row(
-                  crossAxisAlignment: CrossAxisAlignment.center,
-                  children: [
-                    Logo(),
-                  ],
-                ),
-                WelcomeScreen(),
-                LoginPage(),
-                GoogleLogin(),
-                LoginBottom(),
-              ],
-            ),
+    return Scaffold(
+      backgroundColor: Color(0xfff8f9fa), // just add oxff before hexa code
```

Comment:
we shouldn't not use hard coded colors or fonts like this. this causes problems in the long run. instead need to use this always https://flutter.dev/docs/cookbook/design/themes

Reply:
this is used for color and font